.maestro/security_tests/0_all.yaml

```diff
@@ -0,0 +1,14 @@
+appId: com.duckduckgo.mobile.android
+tags:
+    - securityTest
```

Comment:
Do we need this file? 

Since this is tagged with `securityTest` and all the tests (1-7) are also tagged with `securityTest`, does that not mean that everything will run twice in `end-to-end.yml`?

Reply:
You're right - everything would run twice this way. I was using it in testing to speed it up so I wouldn't have to repeatedly run through the onboarding process for each test. I've removed it.